app/(pages)/content/[collection]/edit/_components/Description/TextEditor.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import styles from './DescriptionForm.module.scss';
+import React, { useState } from 'react';
+import ReactQuill from 'react-quill';
+import './quill.snow.css';
```

Comment:
Does next.js complain about this? I'm not sure if we can just import css into components. We could put it as another "global" css file and have this get imported into the root the same way globals.scss is done.

Reply:
I don't think next.js had any issues with it being css actually, but I changed it to scss for our peace of mind along with moving it into the globals folder. I also added a new page under /content route, so you can check out how it looks like.